exporter/otlphttpexporter/otlp.go

```diff
@@ -389,3 +393,27 @@ func (e *baseExporter) logsPartialSuccessHandler(protoBytes []byte, contentType
 	}
 	return nil
 }
+
+// This is a copy from the OTLP HTTP Receiver, i'm not sure the right place to share this logic...
+func newStatusFromMsgAndHTTPCode(errMsg string, statusCode int) *status.Status {
```

Comment:
not sure where this should go, it's the same function as the otlp receiver but I don't think it makes sense to import the receiver from the exporter... Should this go in consumer errors?

Reply:
does what I did look good?

Reply:
Yup